src/coreclr/jit/emitarm64.h

```diff
@@ -664,6 +668,12 @@ inline static bool isFloatReg(regNumber reg)
     return isVectorRegister(reg);
 }
 
+inline static bool isPredicateRegister(regNumber reg)
+{
+    // TODO-SVE: Fix once we add predicate registers
+    return (reg >= REG_FP_FIRST && reg <= REG_FP_LAST);
```

Comment:
This could use the new REG_P0 to REG_P15. And/or define `REG_P_FIRST` and `REG_P_LAST`?

Reply:
This is done in https://github.com/dotnet/runtime/pull/95016